<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PangToolClass</class>
 <widget class="QWidget" name="PangToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PangTool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditFilePath"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFile">
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDir">
         <property name="text">
          <string>选择目录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonTransForm">
         <property name="text">
          <string>执行转换</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PangTool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonFile</sender>
   <signal>clicked()</signal>
   <receiver>PangToolClass</receiver>
   <slot>on_file_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDir</sender>
   <signal>clicked()</signal>
   <receiver>PangToolClass</receiver>
   <slot>on_dir_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonTransForm</sender>
   <signal>clicked()</signal>
   <receiver>PangToolClass</receiver>
   <slot>on_transform_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_file_clicked()</slot>
  <slot>on_dir_clicked()</slot>
  <slot>on_transform_clicked()</slot>
 </slots>
</ui>
